apps/condo/domains/acquiring/components/payments/PaymentsTable.tsx

```diff
@@ -92,8 +106,11 @@ const PaymentsTable: React.FC<IPaymentsTableProps> = ({ billingContext, contexts
     const [dateRange, setDateRange] = useDateRangeSearch('advancedAt', loading)
 
     useEffect(() => {
-        if (!dateRange && appliedFiltersCount < 1 && !search) {
+        if (!hasFilters && shouldApplyDefaultFilter && !isDefaultFilterApplied) {
+            isDefaultFilterApplied = true
             setDateRange(DEFAULT_DATE_RANGE)
+        } else {
+            shouldApplyDefaultFilter = false
```

Comment:
the same.

I think such flags should be individual for every instance

Reply:
No, this is mandatory behavior. Here we need to keep value between table rerenders to prevent setting default filters after all filters will be cleaned.